s3/replication/build_and_test.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+#
+# build-runalltest.sh allows to build all multisite moudles,
+# create virtual environment, populating service files with
+# correct paths, copying to default systemd location and
+# starting background processes and running load transfer tests for
+# s3replicator and s3replicationmanager.
+#
+# Usage : sh ./build_and_test.sh
+# 
+
+# Project root directory
+MULTISITE_ROOT_DIR="$(cd ../../; pwd)"
+echo "Project directory : "$MULTISITE_ROOT_DIR
+
+# Replication directory
+REPLICATION_DIR="$(pwd)"
```

Comment:
Better alternative, see this example https://github.com/Seagate/cortx-s3server/blob/main/rpms/s3/buildrpm.sh#L24-L25

Reply:
done